src/components/confetti/Confetti.tsx

```diff
@@ -0,0 +1,56 @@
+import ReactConfetti from "react-confetti";
+import React, { useEffect } from "react";
+import { styled } from "@storybook/theming";
+import { createPortal } from "react-dom";
+
+interface ConfettiProps
+  extends Omit<React.ComponentProps<typeof ReactConfetti>, "drawShape"> {
+  top: number;
+  left: number;
+  width: number;
+  height: number;
+}
+
+const confettiContainer = document.createElement("div");
+confettiContainer.setAttribute("id", "confetti-container");
+confettiContainer.setAttribute(
+  "style",
+  "position: fixed; top: 0; left: 0; width: 100%; height: 100%; pointer-events: none; z-index: 9999;"
+);
+
+const Wrapper = styled.div<{
+  width: number;
+  height: number;
+  top: number;
+  left: number;
+}>(({ width, height, left, top }) => ({
+  width: `${width}px`,
+  height: `${height}px`,
+  left: `${left}px`,
+  top: `${top}px`,
+  position: "relative",
+  overflow: "hidden",
+}));
+
+export function Confetti({
+  top = 0,
+  left = 0,
+  width = window.innerWidth,
```

Comment:
I don't know we care about this, but this is a one-off setting, which means that it's not responsive. Is that OK? If the user resizes anything in the browser, the confetti will look off e.g. 
<img width="825" alt="image" src="https://github.com/storybookjs/addon-onboarding/assets/1671563/47dba3d3-2216-4e2a-8b4b-a99ef3a1ed12">

Reply:
The positioning of the container is controlled. Whoever controls the container has to also make sure updating it, if positions are changing.